invenio_damap/assets/semantic-ui/js/invenio_damap/DMPAuthButton.js

```diff
@@ -0,0 +1,52 @@
+// This file is part of Invenio-DAMAP
+// Copyright (C) 2023-2024 Graz University of Technology.
+// Copyright (C) 2023-2024 TU Wien.
+//
+// Invenio-DAMAP is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { Button, Icon, Popup } from "semantic-ui-react";
+
+import PropTypes from "prop-types";
+
+// TODO: Make overridable
+// TODO: We could render an extra dialog instead of redirecting. This should be possible when
+// DAMAP offers a way to check between null (no identity) vs empty list (no DMPs created).
```

Comment:
Currently, there is no way of distinguishing between not having DMPs and not being in the system at all DAMAP. The outcome here could be handled the same though: There are no DMPs to display.

DAMAP might have some sort of user management in the future. Just to make such identity checks or linking easier.

Reply:
fyi i'm greatly in favour of having some sort of user management in DAMAP, as it would make future implementations/integrations on/with both systems a lot easier. It's future work which will take some effort, however.